src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -256,6 +256,26 @@ public function deleteAction(Request $request)
 
     /**
      * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function revertDeleteAction(Request $request)
+    {
+        $this->get('doctrine')->getManager()->getFilters()->disable('softdeleteable');
```

Comment:
Are you sure it is? We should enable it explicitly, because someone may extend Sylius and not expect that this filter is disabled.